lib/worker/src/rpc.rs

```diff
@@ -1,4 +1,6 @@
-use crate::{crypto, db, gcs::GcsClient, sql, utils, web3storage::Web3StorageClient};
```

Comment:
nothing interesting in this file, just refactoring to keep up with changes of names and types. this file will be removed when we remove capnp